cmd/api/acrsdk.go

```diff
@@ -49,10 +54,11 @@ func LoginURLWithPrefix(loginURL string) string {
 }
 
 func NewAcrCLIClient(loginURL string) AcrCLIClient {
-	loginURL = LoginURLWithPrefix(loginURL)
+	loginURLPrefix := LoginURLWithPrefix(loginURL)
 	return AcrCLIClient{
-		acrapi.NewWithoutDefaults(loginURL),
-		100,
+		AutorestClient:        acrapi.NewWithoutDefaults(loginURLPrefix),
+		manifestTagFetchCount: 100,
```

Comment:
make a const?

Reply:
Making it a parameter makes sense, but should the error channel buffer size (for the worker) be configurable by the user?